pkg/registry/core/rest/storage_core.go

```diff
@@ -184,15 +184,16 @@ func (c LegacyRESTStorageProvider) NewLegacyRESTStorage(restOptionsGetter generi
 	serviceRest, serviceRestProxy := servicestore.NewREST(serviceRESTStorage, endpointsStorage, podStorage.Pod, serviceClusterIPAllocator, serviceNodePortAllocator, c.ProxyTransport)
 
 	restStorageMap := map[string]rest.Storage{
-		"pods":             podStorage.Pod,
-		"pods/attach":      podStorage.Attach,
-		"pods/status":      podStorage.Status,
-		"pods/log":         podStorage.Log,
-		"pods/exec":        podStorage.Exec,
-		"pods/portforward": podStorage.PortForward,
-		"pods/proxy":       podStorage.Proxy,
-		"pods/binding":     podStorage.Binding,
-		"bindings":         podStorage.Binding,
+		"pods":                     podStorage.Pod,
+		"pods/attach":              podStorage.Attach,
+		"pods/status":              podStorage.Status,
+		"pods/ephemeralcontainers": podStorage.EphemeralContainers,
```

Comment:
only add this if the feature is enabled... we don't even want the rest endpoint registered if the feature is gated